pkgs/build-support/setup-hooks/update-golang-sys-hook.sh

```diff
@@ -0,0 +1,14 @@
+preConfigureHooks+=(_updateGolangSys)
+
+_updateGolangSys() {
+    sed -i '/golang.org\/x\/sys/d' go.mod
+    cat >>go.mod <<EOF
+require (
+    golang.org/x/sys v0.0.0-20220731174439-a90be440212d // indirect
```

Comment:
```suggestion
    golang.org/x/sys v0.1.0 // indirect
```
lets use the recent 0.1.0 tag

Reply:
The problem is that 0.1.0 only supports Go 1.17+, while `awsls` was anchored to `go 1.16` in `go.mod`. Maybe we should rename this to `updateGolangSysHook116` or something?